src/packages/@ncigdc/utils/consoleDebug.tsx

```diff
@@ -1,9 +1,12 @@
-import { IS_DEV } from '@ncigdc/utils/constants';
+import {
+  DEBUGGING,
+  IS_DEV,
+} from '@ncigdc/utils/constants';
 
 type TConsoleDebug = (log: string) => void;
 
 const consoleDebug: TConsoleDebug = log => {
-  if (IS_DEV) {
+  if (IS_DEV || DEBUGGING) {
```

Comment:
This will allow flag-based prod console logging, for better testing and debugging.